common/pde.h

```diff
@@ -0,0 +1,658 @@
+#pragma once
+
+#include <rte_build_config.h>
```

Comment:
We should not rely on DPDK elsewhere except DataPlane.
In the opposite case this means that any utility as `cli` should depended on DPDK and be linked with.